shared/wallets/receive-modal/index.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import * as React from 'react'
+import {Box2, Button, CopyText, Icon, InfoNote, MaybePopup, Text} from '../../common-adapters'
+import {globalMargins, isMobile, platformStyles, styleSheetCreate} from '../../styles'
+
+type Props = {
+  federatedAddress: string,
+  onClose: () => void,
+  stellarAddress: string,
+  username: string,
+}
+
+const ReceiveModal = (props: Props) => (
+  <MaybePopup onClose={props.onClose}>
+    <Box2 direction="vertical" style={containerStyle}>
+      <Icon
+        type={isMobile ? 'icon-wallet-receive-64' : 'icon-wallet-receive-48'}
+        style={{marginBottom: globalMargins.small}}
+      />
+      <Text type="BodySmallSemibold">{props.username}’s wallet</Text>
+      <Text type="Header" style={{marginBottom: globalMargins.medium}}>
+        Receive
+      </Text>
+      <Text
+        type="Body"
+        style={{
```

Comment:
why not have these styles in the stylesheet create below

Reply:
I wasn't sure what the threshold was for moving styles into `styleSheetCreate`. Should we always use that, regardless of the number of styles? For instance, should the single margin declarations be moved too?

Reply:
i think its a lot cleaner if they're all down there

Reply:
Works for me. Thanks!